crates/environ/src/module.rs

```diff
@@ -311,6 +272,149 @@ impl Default for MemoryInitialization {
     }
 }
 
+#[allow(missing_docs)]
+pub enum InitMemory {
```

Comment:
Could we have doc-comments here to describe what each return-code means?

Reply:
Oh I meant to come back to this but I ended up not using anything but `Ok` for testing, so I went ahead and switched this to a `bool`.